src/commands/sync.js

```diff
@@ -1,12 +1,25 @@
 /* @flow */
 
 import Command from '../command'
+import { CLIError } from '../common'
 
 export default class SyncCommand extends Command {
   name = 'sync'
   description = 'Sync upstream changes'
 
-  callback() {
-    console.log('sync', this.projectsRoot)
+  async callback() {
+    const projects = await this.getProjects()
+    if (!projects.length) {
+      throw new CLIError("You don't have any projects to sync")
+    }
+    const promises = []
+    const syncRepo = project => this.spawn('git', ['fetch', '--all'], { stdio: 'inherit', cwd: project.path })
+    projects.map((project) => {
+      promises.push(syncRepo(project))
```

Comment:
Can we inline the `syncRepo` function here please? One liner, v8 inlines it anyway plus it increases readability